charts/multiplanetary/templates/network.yaml

```diff
@@ -0,0 +1,149 @@
+{{ range $.Values.multiplanetary }}
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: {{ .name }}
+  namespace: argocd
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/planetarium/9c-infra.git
+    targetRevision: main
+    path: charts/all-in-one
+    helm:
+      values: |-
+        clusterName: {{ $.Values.clusterName }}
+        logLevel: {{ $.Values.logLevel }}
+
+        global:
+          image:
+            {{- toYaml (default $.Values.global.image .image) | nindent 12 }}
+          appProtocolVersion: {{ default $.Values.global.appProtocolVersion .appProtocolVersion }}
+          genesisBlockPath: {{ default $.Values.global.genesisBlockPath .genesisBlockPath }}
+          trustedAppProtocolVersionSigner: {{ default $.Values.global.trustedAppProtocolVersionSigner .trustedAppProtocolVersionSigner }}
+          peerStrings:
+            {{- toYaml (default $.Values.global.peerStrings .peerStrings) | nindent 12 }}
+          iceServers:
+            {{- toYaml (default $.Values.global.iceServers .iceServers) | nindent 12 }}
+          networkType: {{ default $.Values.global.networkType .networkType }}
+          consensusType: {{ default $.Values.global.consensusType .consensusType }}
+          resetSnapshot: {{ default $.Values.global.resetSnapshot .resetSnapshot }}
+          slackToken: {{ default $.Values.global.slackToken .slackToken }}
+          slackWebHookUrl: {{ default $.Values.global.slackWebHookUrl .slackWebHookUrl }}
+
+        externalSecret:
+          enabled: {{ default $.Values.externalSecret.enabled .externalSecret.enabled }}
+          prefix: "{{ $.Values.clusterName }}/heimdall"
+
+        snapshot:
+          downloadSnapshot: {{ default $.Values.snapshot.downloadSnapshot .snapshot.downloadSnapshot }}
+          fullSnapshot:
+            {{- toYaml (default $.Values.snapshot.fullSnapshot .snapshot.fullSnapshot) | nindent 12 }}
+          partitionReset:
+            {{- toYaml (default $.Values.snapshot.partitionReset .snapshot.partitionReset) | nindent 12 }}
+          partition:
+            {{- toYaml (default $.Values.snapshot.partition .snapshot.partition) | nindent 12 }}
+          slackChannel: {{ default $.Values.snapshot.slackChannel .snapshot.slackChannel }}
+          image:
+            {{- toYaml (default $.Values.snapshot.image .snapshot.image) | nindent 12 }}
+          cfDistributionId: "{{ default $.Values.snapshot.cfDistributionId .snapshot.cfDistributionId }}"
+          aws_access_key_id: "{{ default $.Values.snapshot.aws_access_key_id .snapshot.aws_access_key_id }}"
+          aws_secret_access_key: "{{ default $.Values.snapshot.aws_secret_access_key .snapshot.aws_secret_access_key }}"
+          path: "{{ $.Values.clusterName }}/heimdall"
```

Comment:
should use `snapshot.path` variable

Reply:
done!

Reply:
@Atralupus did you also write the value `snapshot.path`?? i couldnt find

Reply:
no, I didn't... can i write the value like this?

```
    snapshot:
      path: 9c-internal-v2/heimdall
```

Reply:
correct